core/sqf/monitor/linux/monitor.cxx

```diff
@@ -1089,6 +1090,22 @@ int main (int argc, char *argv[])
         tmpptr++;
     }
 
+    // Remove the domain portion of the name if any
```

Comment:
There are actually four places and unfortunately in the scope of different processes. I will keep this in mind as there is some utility code that could perhaps be refactored to contain functions like this.